src/main/scala/io/druid/indexer/spark/SparkDruidIndexer.scala

```diff
@@ -197,7 +196,8 @@ object SparkDruidIndexer extends Logging {
         new DateBucketAndHashPartitioner(
           dataSchema.getDelegate.getGranularitySpec.getSegmentGranularity,
           hashToPartitionMap,
-          optionalDims
+          optionalDims,
+          Option(dataSchema.getDelegate.getParser.getParseSpec.getDimensionsSpec.getDimensionExclusions.asScala.toSet)
```

Comment:
use `Some(xxx)` to indicate this has a value, `Option(...)` is typically only used when you want to map nulls to None.